scrapinghub/client/utils.py

```diff
@@ -140,6 +132,46 @@ def list(self, *args, **kwargs):
         return list(self.iter(*args, **kwargs))
 
 
+class _MappingProxy(_ProxyBase):
+
+    def __init__(self, cls, client, key):
+        super(_MappingProxy, self).__init__(cls, client, key)
+        methods = ['ignore_fields', 'expire', 'save', 'liveget']
+        self._proxy_methods(methods)
+        self.iterkeys = self.__iter__
```

Comment:
Not sure about pros, we can't proxy special methods with underscores and totally there're only 4 methods listed here above + `iterkeys`, I have doubts that blacklisting will simplify anything. But probably I didn't get the whole idea, could you explain pls?